airbyte-integrations/connectors/source-google-sheets/Dockerfile

```diff
@@ -34,5 +34,5 @@ COPY google_sheets_source ./google_sheets_source
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.2.18
+LABEL io.airbyte.version=0.2.19
```

Comment:
State isn't used by google-sheets, but for consistency, we might also want to update the overwritten `read()` method to the new signature:
```
    def read(
        self,
        logger: logging.Logger,
        config: Mapping[str, Any],
        catalog: ConfiguredAirbyteCatalog,
        state: Union[List[AirbyteStateMessage], MutableMapping[str, Any]] = None
    ) -> Generator[AirbyteMessage, None, None]:
```